examples/game_of_life/_helpers.py

```diff
@@ -82,21 +48,10 @@ def get_active_pin_map_id(self) -> str:
         Returns:
             The resource id of the pin map that is registered to the pin map service.
         """
-        return self._sequence_context.Engine.TemporaryGlobals.GetValString(
```

Comment:
FYI, the `Execution.RunTimeVariables` update and `set_active_pin_map` API removal were missed for the non-instruments in the earlier PRs. Hence, I'm accommodating those changes as well in this PR.